Gemfile

```diff
@@ -24,6 +24,7 @@ gem 'rails-controller-testing', '~> 0.1'
 gem 'govuk_ab_testing', '~> 2.0'
 gem 'htmlentities', '4.3.4'
 gem 'statsd-ruby', '1.3.0', require: 'statsd'
+gem 'sprockets-rails'
```

Comment:
government-frontend is using sprockets-rails 3.1.1 at the moment, so I'll have to add an explicit version